alchemist/alchemist-incarnation-biochemistry/src/test/java/it/unibo/alchemist/test/TestBioRect2DEnvironmentNoOverlap.java

```diff
@@ -0,0 +1,761 @@
+package it.unibo.alchemist.test;
```

Comment:
if such numbers have a meaning, then yes, use a private constant static field. E.g. if you expect the distance between two cells to be 134, you can declare something like:
private static final EXPECTED_DISTANCE = 134

If you are unsure, ask me :)